hack/env.sh

```diff
@@ -13,6 +15,8 @@ else
         [ -z $SRIOV_INFINIBAND_CNI_IMAGE ] && echo "SRIOV_INFINIBAND_CNI_IMAGE is empty but SKIP_VAR_SET is set" && exit 1
         # check that OVS_CNI_IMAGE is set to any value, empty string is a valid value
         [ -z ${OVS_CNI_IMAGE+set} ] && echo "OVS_CNI_IMAGE is empty but SKIP_VAR_SET is set" && exit 1
+        # check that RDMA_CNI_IMAGE is set to any value, empty string is a valid value
+        [ -z ${$RDMA_CNI_IMAGE+set} ] && echo "RDMA_CNI_IMAGE is empty but SKIP_VAR_SET is set" && exit 1
```

Comment:
please do something like we did for the ovs-cni

```
# ensure that OVS_CNI_IMAGE is set, empty string is a valid value
OVS_CNI_IMAGE=${OVS_CNI_IMAGE:-}
```

Reply:
done